src/constructors/regular.c

```diff
@@ -420,6 +420,81 @@ igraph_error_t igraph_tree(igraph_t *graph, igraph_integer_t n, igraph_integer_t
     return IGRAPH_SUCCESS;
 }
 
+igraph_error_t igraph_tree_from_parent_vector(igraph_t *graph, igraph_vector_int_t* parents,
+                igraph_tree_mode_t type) {
+
+    igraph_vector_int_t edges = IGRAPH_VECTOR_NULL;
+    igraph_vector_int_t visited = IGRAPH_VECTOR_NULL;
```

Comment:
I suggest not to initialize to values you will never use. This tends to interfere with static analysis tools.